test/Microsoft.Azure.Webjobs.Extensions.Storage.UnitTests/DefaultStorageCredentialsValidatorTests.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net.Http;
+using System.Reflection;
+using Microsoft.WindowsAzure.Storage;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Host.UnitTests.Executors
+{
+    public class DefaultStorageCredentialsValidatorTests
+    {
+        [Fact]
+        public void UseReflectionToAccessIsDevStoreAccountFromCloudStorageAccount()
+        {
+            var isDevStoreAccountProperty = typeof(CloudStorageAccount).GetProperty("IsDevStoreAccount", BindingFlags.NonPublic | BindingFlags.Instance);
+            Assert.NotNull(isDevStoreAccountProperty);
+        }
+
+        [Theory(Skip = "Missing StorageCredentialsValidator")]
```

Comment:
I see - you moved the tests. What's the plan for the validator? Why don't we validate anymore?